hack/preload-images/upload.go

```diff
@@ -67,7 +67,7 @@ func getVersionsFromFilename(filename string) (string, string) {
 	preloadVersion := parts[3]
 	k8sVersion := parts[4]
 	// this check is for "-rc" and "-beta" versions that would otherwise be stripped off
-	if len(parts) == 9 {
+	if len(parts) >= 9 && parts[5] != "cri" {
```

Comment:
Is it possible that `parts[5]` could be `cri-o` instead of just `cri`?

Reply:
No, the string is being split on `-`, so `cri` would be the value of `parts[5]`